include/z64map_mark.h

```diff
@@ -0,0 +1,27 @@
+#ifndef _Z64MAP_MARK_H_
```

Comment:
Imo there isn't really a need to do a separate header for just these few things right now.
We'll probably have a dedicated header for this system once we restructure everything, but there is a good chance its scope will be different and that it won't be in `include/`, so I suggest keeping it in `z64.h` for now.

Reply:
if z64 is being split, i see no reason not to split it in pieces as we go if you have a good sense of how to split it.

Reply:
would normally agree to splitting it out for larger systems that get references alot. but romans last point here is the main thing `but there is a good chance its scope will be different and that it won't be in include/, so I suggest keeping it in z64.h for now.`